lib/Sabberworm/CSS/Parsing/ParserState.php

```diff
@@ -216,7 +217,8 @@ public function consumeUntil($aEnd, $bIncludeEnd = false, $consumeEnd = false, a
 		$start = $this->iCurrentPosition;
 
 		try {
-			while (($char = $this->consume(1)) !== '') {
+			while (true) {
+				$char = $this->consume(1);
```

Comment:
Maybe test for `isEnd` and handle the `in_array(self::EOF, $aEnd)` case gracefully instead of relying on the exception handling here.